app/assets/stylesheets/_main.scss

```diff
@@ -0,0 +1,20 @@
+.site-main {
+  padding: 2em 0; // should this be in here?
```

Comment:
Oh, no, it can be removed now!

It's a remainder from back when the padding was defined in `_shared.scss`, which just felt a bit wrong, as all the other rules in `_shared.scss` were about background-color and nothing else.